### PR TITLE
Tbolt/2822 money formatting issue

### LINKED ISSUE #2822: Viewing a field with money is only formatted when exiting the field, but not when access the field again
After entering funding in a field, the commas are applied to fields involving $ values.

How to view the bug:

1. Enter data into a money field. Move to another field.

2. After data entry:

3. After collapsing the data entry section and reopening it / reloading the APD, the formatting is lost:

Collapse/Reopen:

4. (Alternative view to reproduce) Load the seeded APD and look at the Results of Previous Activities / Actual Expenditures section:

### This task is done when...
 - [ ] Fields with money are properly formatted no matter when a user is viewing 
 - [ ] Someone else has reviewed the code
- [ ] Design Signs Off
- [ ] Product Signs Off

### PR DESCRIPTION
**Pull request by tbolt**
Resolves #2822 

### This pull request changes...

- Updates fields that use a currency mask (ie. changing $1234 -> $1,234) to keep the mask applied when closing and re-opening the parent components (or simply re-loading the APD)

### This pull request is ready to merge when...

- [x] Develop has updated automated tests (and all tests are passing)
- [x] This code has been reviewed by someone other than the original author
- [x] QA has manually tested and created bugs for any issues found (beyond minor fixes)
- [x] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented
- [x] The change has been documented
- [x] Associated OpenAPI documentation has been updated
- [x] Changelog is updated as appropriate

### This feature is done when...

- [ ] QA has verified the accessibility and functionality related to the change
- [ ] Design has approved the experience
- [x] Product has approved the experience

### CODE REVIEW

**Review by thetif — APPROVED** [review]
looks good to me

File: web/src/actions/auth.js [hunk 1]
```diff
@@ -135,7 +135,7 @@ const setupTokenManager = () => (dispatch, getState) => {
       // called when the token is closer to expiring
```
Comment (eStar386) [inline web/src/actions/auth.js:135]: @tbolt Why was this code change included in the money formatting issue? This doesn't appear to be related.

Comment (tbolt) [reply web/src/actions/auth.js:135]: It's formatting updates based on our `prettier js` rules

**Review by eStar386 — APPROVED** [review]
Passes functionality and accessibility testing. 
1. Load the seeded APD and look at the Results of Previous Activities / Actual Expenditures section. 
Result: The mask remains on the pdf/ user views.

**Review by beparticular — APPROVED** [review]

**Review by NAretakis — APPROVED** [review]
works as advertised, good to go!

**Pull request merged**